srcpkgs/epson-inkjet-printer-escpr2/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'epson-inkjet-printer-escpr2'
+pkgname=epson-inkjet-printer-escpr2
+version=1.1.15
+revision=1
+archs="x86_64 i686 aarch64 armv7l"
+create_wrksrc=no
+build_wrksrc=${pkgname}-${version}
+build_style=configure
+configure_script=./configure
+configure_args="--prefix=/usr --with-cupsfilterdir=/usr/lib/cups/filter --with-cupsppddir=/usr/share/ppd"
+hostmakedepends="automake rpmextract tar"
+makedepends="cups-devel ghostscript-devel"
+short_desc="Epson Inkjet Printer Driver 2 (ESC/P-R) for Linux"
+maintainer="André Cerqueira <acerqueira021@gmail.com>"
+license="custom:Proprietary"
+homepage="http://download.ebz.epson.net/dsc/search/01/search/?OSC=LX"
+distfiles="https://download3.ebz.epson.net/dsc/f/03/00/11/56/43/b1a6f1193e073c4c049da9404806c61052b62419/${pkgname}-${version}-1lsb3.2.src.rpm"
+checksum=2eb8f1655de4f7189ec18d94f581112e2ff00b05e64a44c24bbb0524fafdd728
+restricted=yes
+
+post_extract() {
+	tar xvf ${pkgname}-${version}-1lsb3.2.tar.gz
```

Comment:
Use `bsdtar`

Reply:
Any reason in specific ?

Is bsdtar syntax correct ?
```suggestion
	bsdtar -x ${pkgname}-${version}-1lsb3.2.tar.gz
```

Reply:
`bsdtar xf FILE` should work. bsdtar is in default build environment, so tar could be avoided in hostmakedepends.

Reply:
Used , thanks for the explanation !